docs/advanced/sphinx.md

```diff
@@ -169,11 +169,11 @@ You can also define TeX macros for a specific file by introducing them at the be
 The commands can be used inside a `math` directive, `$$` or inline `$`, for example
 ```md
 $$
-A = \bmat 1 & 1 \\ 2 & 1\\ 3 & 2 \emat,\ b=\bmat 2\\ 3 \\ 4\emat,\ \gamma = 0.5
+A = \bmat{} 1 & 1 \\ 2 & 1\\ 3 & 2 \emat{},\ b=\bmat{} 2\\ 3 \\ 4\emat{},\ \gamma = 0.5
```

Comment:
@chrisjsewell this isn't usually how I would write this in latex. Typically macro substitutions I have used in the past don' thave `{}` -- is there a reason for adding these?

Reply:
Yes because what your doing is wrong lol. Look at the current documentation, and see how `\bmat` "eats up" the preceding 1: https://jupyterbook.org/advanced/sphinx.html
You need to close them to stop any arguments being passed

Reply:
I see -- so the `array` environment is eating the `1` as an option -- gotcha. Macros I have used in the past have been more direct options (like operator substitutions). You're right!.

Reply:
FYI https://www.nongnu.org/chktex/ is a good linter for latex, which will complain about these "open" commands